src/components/MenuItem.tsx

```diff
@@ -49,14 +49,14 @@ type UnresponsiveProps = {
 
 type IconProps = {
     /** Flag to choose between avatar image or an icon */
-    iconType: typeof CONST.ICON_TYPE_ICON;
+    iconType?: typeof CONST.ICON_TYPE_ICON;
 
     /** Icon to display on the left side of component */
     icon: IconAsset;
 };
 
 type AvatarProps = {
-    iconType: typeof CONST.ICON_TYPE_AVATAR | typeof CONST.ICON_TYPE_WORKSPACE;
+    iconType?: typeof CONST.ICON_TYPE_AVATAR | typeof CONST.ICON_TYPE_WORKSPACE;
```

Comment:
Is there any specific reason this is optional?

Reply:
Yes, because `iconType` should be optional, there is even a default value for it inside the component.

Reply:
Yes, you can also see that `iconType` isn't passed as a prop to the MenuItem [in this file](https://github.com/Expensify/App/pull/33242/files#diff-52cad1bd9d75e9f709334465fb0ff19f0d0b09e9fda5faffec28b5c547c65110R154).

Reply:
thanks for explaining